lstchain/scripts/lstchain_r0_to_r0g.py

```diff
@@ -76,15 +69,13 @@ def main():
     for name in input_stream_names:
         input_streams.append(protozfits.File(name, pure_protobuf=True))
 
-    is_evb6_data = False
     try:
-        num_pixels = input_streams[0].CameraConfig[0].num_pixels
-        num_samples = input_streams[0].CameraConfig[0].num_samples
-    except:
-        is_evb6_data = True
         num_pixels = input_streams[0].CameraConfiguration[0].num_pixels
         num_samples = input_streams[0].CameraConfiguration[
             0].num_samples_nominal
+    except:
+        logging.error('CameraConfiguration not found! Is this CTAR1 data?')
```

Comment:
CTAR1 is an old name that will disappear soon, we should use `R1v1` (official CTAO / new EVBv6 protobuf message) vs. `ProtoR1` (old EVBv5 protobuf messages)

Reply:
I think the data you uploaded are the 1-gain test data... (20231218) tests are failing because of that...

Reply:
My bad, sorry, will upload files from another date... Do you know one by heart that would work?

Reply:
20231214/LST-1.1.Run16102.0010.fits.fz

Reply:
Thanks a lot!